OSDK.toml

```diff
@@ -40,7 +40,8 @@ qemu.args = "$(./tools/qemu_args.sh microvm)"
 
 [scheme."iommu"]
 supported_archs = ["x86_64"]
-qemu.args = "$(./tools/qemu_args.sh iommu)"
+# OVMF provides the firmware necessary for supporting IOMMU in the default EFI boot mode.
+qemu.args = "$(./tools/qemu_args.sh iommu -ovmf)"
```

Comment:
According to this comment, IOMMU requires OVMF. Then, why not let `qemu_args` enable OVMF automatically?

Reply:
My first intention is both `iommu` and `-ovmf` are readily and seperately script parameters that can be used directly and conveniently.
Integrating OVMF arguments with IOMMU directly without exposing `-ovmf` is also feasible, and maybe better. I will modify it.

Reply:
After I looked carefully at `OSDK.toml` and `qemu_args.sh`, I may have different opinions. We only utilize `qemu_args.sh` in four cases in `OSDK.toml`:

```
...

[qemu]
args = "$(./tools/qemu_args.sh normal -ovmf)"

...

[test.boot]
method = "qemu-direct"

[test.qemu]
args = "$(./tools/qemu_args.sh test)"

...

[scheme."microvm"]
boot.method = "qemu-direct"
build.strip_elf = true
qemu.args = "$(./tools/qemu_args.sh microvm)"

...

[scheme."iommu"]
supported_archs = ["x86_64"]
qemu.args = "$(./tools/qemu_args.sh iommu -ovmf)"

...
```

It is showed that we must enable OVMF in the default EFI boot mode ("normal" and "iommu"). And in the "qemu-direct" boot mode, OVMF is no longer needed and disabled.

So in this PR, we either just write `./tools/qemu_args.sh iommu -ovmf` without any comments. It indicates this scheme is similar to the default `./tools/qemu_args.sh normal -ovmf` that we use the EFI boot mode. Or we need to hide OVMF-related commands into `qemu_args.sh` without exposing `-ovmf`.

Reply:
The changes in https://github.com/asterinas/asterinas/pull/1553 appear to address the issue I mentioned earlier. This PR can be based on that.

Reply:
Yep, #1553 , though intended to solve a different problem, can provide a better solution for this.

Reply:
So let's merge #1553 first, then rebase this PR on it?

Reply:
Yes, I think so.

Reply:
#1533 merged

Reply:
Revised and pushed